helpers/data.js

```diff
@@ -71,12 +71,59 @@ module.exports = {
 	},
 
 	getTemplateCode: function(filePath) {
+		function highlightTemplateCode(content) {
```

Comment:
Can we have this function either in the main scope (like `getTemplateCode` or `getUsedPartialsInTemplate`)?